RecoEgamma/EgammaTools/python/calibratedEgammas_cff.py

```diff
@@ -1,9 +1,11 @@
 import FWCore.ParameterSet.Config as cms
 
```

Comment:
@jainshilpi  you need to add 
from Configuration.Eras.Modifier_run2_miniAOD_devel_cff import run2_miniAOD_devel